### PR TITLE
LPS-131312 | master

### PR DESCRIPTION
**Pull request by kyle-miho**
https://issues.liferay.com/browse/LPS-131312

### CODE REVIEW

**Comment by kyle-miho** [comment]
ci:test:sf

**Comment by kyle-miho** [comment]
passed here: https://testray.liferay.com/home/-/testray/runs?testrayBuildId=1550521657 Only failing is failing upstream, rolling restart

File: portal-web/test/functional/com/liferay/portalweb/macros/Clustering.macro [hunk 1]
```diff
@@ -14,13 +14,11 @@ definition {
...
-	macro deployJarOnSpecificServer {
-		var leadingPortNumber = StringUtil.replace("${nodePort}", "080", "");
-		var bundleIndex = ${leadingPortNumber} - 8;
-
-		FileInstall.deployFileOnServer(
-			bundleIndex = "${bundleIndex}",
-			liferayDependencyJars = "${liferayDependencyJars}");
+	macro deployClusteringJar {
+		FileInstall.gradleDeployOnServer(
+			jarName = "com.liferay.cluster.test.module.jar",
```
Comment (vicnate5) [inline portal-web/test/functional/com/liferay/portalweb/macros/Clustering.macro:19]: @kyle-miho This variable needed?

**Review by vicnate5 — COMMENTED** [review]
Please resend with module fully renamed (package, class names, bnd). Run SF locally.

Rename `gradleDeployOnServer` to `deployModuleOnServer`. This might belong in the cluster macro, I don't think it makes sense in the FileInstall macro unless we also use it for file install testing.

**Pull request closed without merging**